pyproject.toml

```
@@ -53,7 +53,7 @@ mteb = "mteb.cli:main"
 
 [project.optional-dependencies]
 dev = ["ruff==0.6.4", # locked so we don't get PRs which fail only due to a lint update
-"pytest", "pytest-xdist", "pytest-coverage"]
+"pytest", "pytest-xdist", "pytest-coverage", "iso639"]
```

Yea but if it breaks it still documents a version of the dependency that likely worked.

```suggestion
"pytest", "pytest-xdist", "pytest-coverage", 
"iso639>=0.1.4" # used for tests/scripts/test_generate_model_meta.py
]
```